### ISSUE TITLE
[TRANSLATIONS] - Hindi

### ISSUE THREAD

**Issue by AdityaGarg00** [microsoft/Data-Science-For-Beginners#146]
- [ ] Base README.md
- [ ] Quizzes
- [ ] Introduction base README
   - [ ] Defining Data Science README
   - [ ] Defining Data Science assignment
   - [ ] Ethics README
   - [ ] Ethics assignment
   - [ ] Defining Data README
   - [ ] Defining Data assignment
   - [ ] Stats and Probability README
   - [ ] Stats and Probability assignment
- [ ] Working with Data base README
   - [ ] Relational Databases README
   - [ ] Relational Databases assignment
   - [ ] Non-Relational Data README
   - [ ] Non-Relational Data assignment
   - [ ] Python README
   - [ ] Python assignment
   - [ ] Data Preparation README
   - [ ] Data Preparation assignment
- [ ] Vizualization base README
   - [ ] Quantities README
   - [ ] Quantities assignment
   - [ ] Distributions README
   - [ ] Distributions assignment
   - [ ] Proportions README
   - [ ] Proportions assignment
   - [ ] Relationships README
   - [ ] Relationships assignment
   - [ ] Meaningful Visualizations README
   - [ ] Meaningful Visualization assignment
- [ ] Data Science Lifecycle base README
   - [ ] Introduction README
   - [ ] Introduction assignment
   - [ ] Analyzing README
   - [ ] Analyzing assignment
   - [ ] Communication README
   - [ ] Communication assignment
- [ ] Data Science in the Cloud base README
   - [ ] Intro README
   - [ ] Intro assignment
   - [ ] Low Code README
   - [ ] Low Code  assignment
   - [ ] Azure README
   - [ ] Azure  assignment
- [ ] Data Science in the Wild base README
   - [ ] Real World Examples README
   - [ ] Real World Examples assignment

**Comment by joshi-kaushal** [comment]
I'd love to work on this one...

**Comment by subh-chaturvedi** [comment]
I would also love to help with this, can I work with this too?

**Comment by jlooper** [comment]
I recommend starting a discussion in the discussion boards to divide up the work

**Comment by paladique** [comment]
@AdityaGarg00 @subh-chaturvedi @joshi-kaushal  check out this thread here to discuss  #148

**Comment by imsushant12** [comment]
I would like to help with this, can I work with this too?

**Comment by paladique** [comment]
> I would like to help with this, can I work with this too?

@imsushant12 absolutely! Feel free to hop in on that discussion thread in #148

**Comment by imsushant12** [comment]
> > I would like to help with this, can I work with this too?
> 
> @imsushant12 absolutely! Feel free to hop in on that discussion thread in #148

Thankyou 🙌

**Comment by Heril18** [comment]
> **Want to contribute?** Use the discussion board thread to see which sections are currently being worked on and what's left: #148
> 
> * [ ]  Base README.md
> * [ ]  Quizzes
> * [x]  Introduction base README
>   
>   * [ ]  Defining Data Science README
>   * [ ]  Defining Data Science assignment
>   * [x]  Ethics README
>   * [ ]  Ethics assignment
>   * [ ]  Defining Data README
>   * [ ]  Defining Data assignment
>   * [ ]  Stats and Probability README
>   * [ ]  Stats and Probability assignment
> * [x]  Working with Data base README
>   
>   * [ ]  Relational Databases README
>   * [ ]  Relational Databases assignment
>   * [ ]  Non-Relational Data README
>   * [ ]  Non-Relational Data assignment
>   * [ ]  Python README
>   * [ ]  Python assignment
>   * [ ]  Data Preparation README
>   * [ ]  Data Preparation assignment
> * [ ]  Vizualization base README
>   
>   * [ ]  Quantities README
>   * [ ]  Quantities assignment
>   * [ ]  Distributions README
>   * [ ]  Distributions assignment
>   * [ ]  Proportions README
>   * [ ]  Proportions assignment
>   * [ ]  Relationships README
>   * [ ]  Relationships assignment
>   * [ ]  Meaningful Visualizations README
>   * [ ]  Meaningful Visualization assignment
> * [ ]  Data Science Lifecycle base README
>   
>   * [x]  Introduction README
>   * [ ]  Introduction assignment
>   * [ ]  Analyzing README
>   * [ ]  Analyzing assignment
>   * [ ]  Communication README
>   * [ ]  Communication assignment
> * [ ]  Data Science in the Cloud base README
>   
>   * [ ]  Intro README
>   * [ ]  Intro assignment
>   * [ ]  Low Code README
>   * [ ]  Low Code  assignment
>   * [ ]  Azure README
>   * [ ]  Azure  assignment
> * [ ]  Data Science in the Wild base README
>   
>   * [ ]  Real World Examples README
>   * [ ]  Real World Examples assignment

**Comment by ParnaviKulkarni** [comment]
Is it still open can I work on this?

**Comment by paladique** [comment]
@ParnaviKulkarni absolutely! Feel free to hop in and tell others what you plan to work on

**Comment by shivanithorve01** [comment]
Hey can I contribute ?

**Comment by AdityaGarg00** [comment]
Yea Sure Go ahead @shivanithorve01

**Comment by DhanyaHegde01** [comment]
I would love to work on Data Science in real-world examples Readme.

**Comment by iamshanawaz** [comment]
I would love to contribute..

**Comment by AdityaGarg00** [comment]
@iamshanawaz Please go ahead and contribute to the remaining files 😃

**Comment by japneetsingh035** [comment]
Can I contribute to this project?

**Comment by paladique** [comment]
> Can I contribute to this project?

Absolutely! Check out the list at the top to see what's remaining 🙂

**Comment by SachinDabhade** [comment]
As I am from Data Science background, I would love to contribute in this repository.

**Comment by abhi1e9-1e10** [comment]
I love contributing . If you want, I can work on this issue .

**Comment by AdityaGarg00** [comment]
That's great @abhi1e9-1e10  and @SachinDabhade you can start contributing on the left ones and if you have any concern you can message here .😃😃

**Comment by anveshajain19** [comment]
I also want to contribute  , Can you assign me  this issue .

**Comment by jlooper** [comment]
We do not assign issues; rather I encourage you all to go to the Discussion boards and organize the work

**Comment by SachinDabhade** [comment]
I want to work on "Defining Data README" and "Defining Data Science README"

**Comment by bhumikatewary** [comment]
Hi, I am bhumikatewary, a contributor to OpenForce 2022. I would like to work on this issue. I would be making a PR as soon as I am done with resolving the issue. Thank you

**Comment by bhumikatewary** [comment]
Please review my updates on  [TRANSLATIONS] Hindi - Defining Data Science README #359

**Comment by SachinDabhade** [comment]
Sorry Bhumika but this translation is already taken by me... and I have made pull request also. So sorry about it but there are many contribution remaining. Keep trying

**Comment by jlooper** [comment]
hi, I recommend you all gather here https://github.com/microsoft/Data-Science-For-Beginners/discussions/148 and organize a checklist of who's translating which files

**Comment by bhumikatewary** [comment]
I would be glad if I could get guidance on which repos I can contribute in since most of them are already taken

**Comment by gargabhishek100** [comment]
I would like to work on [01-defining-data-science](https://github.com/microsoft/Data-Science-For-Beginners/tree/main/1-Introduction/01-defining-data-science)! Is it is still unpicked?

**Comment by AdityaGarg00** [comment]
This file is already taken @gargabhishek100

**Comment by Saur2158** [comment]
Hey can I also be a part of the project  ? I would really love to help

**Comment by vedant-z** [comment]
@AdityaGarg00 I would like to work on this 🙂

**Comment by AdityaGarg00** [comment]
Sure @Saur2158 and @vedant-z Go ahead you can start contributing just mention the files you start contributing to so that everyone could know that you are contributing to that particular file.

**Comment by vedant-z** [comment]
@AdityaGarg00  Actually I was searching for thr files yet to be translated but I couldn't find one .
If you can point out any particular one it would be of great help.

Thanks

**Comment by AshishG97** [comment]
I would love to contribute to this issue as Hindi is my Native language and I use Hindi in my day-to-day life.

**Comment by charaf19** [comment]
I would love to contribute..

**Comment by AdityaGarg00** [comment]
@vedant-z  just translate the unticked files mentioned in the issue itself but do mention the file whatever you are going to translate .

**Comment by shivam141102** [comment]
Hi @AdityaGarg00, @paladique this will be my first open source contribution, really excited for it, can I please work on the following:
-Data Science in the Cloud base
     -Low Code assignment README

**Comment by AdityaGarg00** [comment]
> 

Yes sure, really glad to hear that @shivam141102 . 

P.S. You can contribute to any file without asking us.

**Comment by sourabhligade** [comment]
Hey, is this still open .I'd like to contribute

**Comment by 1e9abhi1e10** [comment]
> Hey, is this still open .I'd like to contribute

@sourabhligade You can contribute to any file without asking.

**Comment by Daksh3468** [comment]
I have translated  Defining Data Science README file in Hindi. and i am attaching it here , please assign me this so i can contribute in this.
[Defining Data Science README.txt](https://github.com/microsoft/Data-Science-For-Beginners/files/12785498/Defining.Data.Science.README.txt)

**Comment by Daksh3468** [comment]
I have translated Defining Data Science assignment file in Hindi. and i am attaching it here , please assign me this so i can contribute in this.
[Defining Data Science assignment.txt](https://github.com/microsoft/Data-Science-For-Beginners/files/12785570/Defining.Data.Science.assignment.txt)

**Comment by Harshupadhyay221** [comment]
I would also like to work on Data Science in the Wild base README- real world examples assignment...Please assign it to me

**Comment by leestott** [comment]
Translations are now automatically completed using coop translator
